### PR TITLE
MTM-44072 Doc for measurements and events on processing uplink

### PR DESCRIPTION
**Pull request by ashhSAG93**
The goal of the PR is to document the measurements and events created on processing of the uplink message like position, signal strength and spreading factor

### CODE REVIEW

File: content/protocol-integration/lora-actility-bundle/uplink-message.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+weight: 50
+title: Standard measurements and events created while processing the uplink message
+layout: redirect
+---
+
+When an uplink message is sent to the device managed object, the following measurements/events are created:
```
Comment (bbyreddy) [inline content/protocol-integration/lora-actility-bundle/uplink-message.md:7]: - [ ] On the receipt of an uplink message, the Cumulocity Platform creates the following measurements and events.

File: content/protocol-integration/lora-actility-bundle/uplink-message.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+- **Position** - The position of the device is captured by considering the latitude, longitude, altitude and accuracy in the c8y_Position fragment of the device managed object. Also a position update event is created.
```
Comment (bbyreddy) [inline content/protocol-integration/lora-actility-bundle/uplink-message.md:9]: - [ ] c8y_Position fragment is updated to capture the latitude, longitude, altitude and accuracy information of the device.

File: content/protocol-integration/lora-actility-bundle/uplink-message.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+- **Spreading factor** - The spreading factor sent in the uplink message is captured in the c8y_SpreadingFactor fragment of the device Managed Object.
```
Comment (bbyreddy) [inline content/protocol-integration/lora-actility-bundle/uplink-message.md:10]: - [ ] c8y_SpreadingFactor fragment of the device managed object is updated to capture the Spreading Factor of the device.

File: content/protocol-integration/lora-actility-bundle/uplink-message.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+- **Signal strength** - A signal strength measurement is created, by considering the RSSI and SNR values.
```
Comment (bbyreddy) [inline content/protocol-integration/lora-actility-bundle/uplink-message.md:11]: - [ ] c8y_SignalStrength fragment of the device managed object is updated to capture the RSSI and SNR values of the device signal.

File: content/protocol-integration/lora-actility-bundle/uplink-message.md (hunk 1 above)
Comment (ashhSAG93) [reply content/protocol-integration/lora-actility-bundle/uplink-message.md]: Have made the changes in the latest commit

**Review by BeateRixen — APPROVED** [review]

File: content/protocol-integration/lora-actility-bundle/overview.md [hunk 5]
```diff
@@ -21,6 +21,7 @@ The following sections describe how to:
 * [Configure your ThingPark account credentials](#configure-credentials) in {{< product-c8y-iot >}}.
 * [Create device protocols](#create-device-protocols) for all devices.
 * [Register devices](#register-device) and visualize Actility payload using {{< product-c8y-iot >}}.
+* [Standard measurements and events created while processing the uplink message](#uplink-message)
```
Comment (BeateRixen) [inline content/protocol-integration/lora-actility-bundle/overview.md]: This does not fit the heading.

"The following sections describe how to:"

Comment (ashhSAG93) [reply content/protocol-integration/lora-actility-bundle/overview.md]: Removed it in the latest commit

File: content/protocol-integration/lora-actility-bundle/uplink-message.md [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+weight: 50
+title: Uplink message processing
+layout: redirect
+---
+
+On the receipt of an uplink message, the {{< product-c8y-iot >}} platform creates the following measurements and events, and updates the corresponding device managed object.
+
+- **Unprocessed data** - An event of type <code>c8y_ActilityUplinkRequest</code> (this type is based on the event.uplink.type set in the configuration file) is created with the unprocessed data. 
+- **Position** - The <code>c8y_Position</code> fragment of the device managed object is updated to capture the latitude, longitude, altitude and accuracy information of the device. Also, an event is created with the position information.
+- **Spreading factor** - The >code>c8y_SpreadingFactor</code> fragment of the device managed object is updated to capture the spreading factor of the device.
```
Comment (MWindel) [inline content/protocol-integration/lora-actility-bundle/uplink-message.md]:
```suggestion
- **Spreading factor** - The <code>c8y_SpreadingFactor</code> fragment of the device managed object is updated to capture the spreading factor of the device.
```

**Review by BeateRixen — APPROVED** [review]

**Review by MWindel — APPROVED** [review]

**Pull request merged**